process/sync/baseSync.go

```diff
@@ -240,9 +259,55 @@ func (boot *baseBootstrap) loadNotarizedBlocks(blockFinality uint64,
 		}
 	}
 
+	if currentNonce <= blockFinality {
+		return process.ErrBoostrapFromStorage
+	}
+
+	for i := currentNonce + 1; i <= highestNonceInStorer; i++ {
+		boot.cleanupNotarizedStorage(i, hdrNonceHashDataUnit)
+	}
+
+	return nil
+}
+
+func (boot *baseBootstrap) cleanupNotarizedStorage(nonce uint64, hdrNonceHashDataUnit dataRetriever.UnitType) {
+	errNotCritical := boot.removeNotarizedBlockHeader(nonce, hdrNonceHashDataUnit)
+	if errNotCritical != nil {
+		log.Info(fmt.Sprintf("remove notarized block header with nonce %d: %s\n", nonce, errNotCritical.Error()))
```

Comment:
log.debug ?

Reply:
It can stay like this, as bootstrap from storage is not an frequently event and when we do this we need these info if there would be some problems.